lib/src/sv_tlv.c

```diff
@@ -913,7 +913,7 @@ decode_signature(signed_video_t *self, const uint8_t *data, size_t data_size)
   svrc_t status = SV_UNKNOWN_FAILURE;
 
   SV_TRY()
-    SV_THROW_IF(version == 0, SV_INCOMPATIBLE_VERSION);
+    SV_THROW_IF(version > 0 && version < 3, SV_OK);
```

Comment:
This will not work. This will return `SV_OK` all the time. You should provide an error message, not an everything-is-fine-message